moondeck/scenario/run_live_scenario.py

```
@@ -461,6 +474,25 @@ def run_scenario(client: Client, scenario_path: Path, settle_s: float = 1.5,
+                path_ = step.get("path")
+                body = step.get("value", "")
+                if not path_:
+                    print(f"  WRITE {step_name} — missing path, skipped")
+                    step_result["status"] = "skipped"
+                else:
+                    try:
+                        resp = client.post_text(f"/api/file?path={urllib.parse.quote(path_)}", body)
+                        ok = bool(resp.get("ok")) if isinstance(resp, dict) else True
+                        step_result["status"] = "ok" if ok else "error"
+                        print(f"  WRITE {path_} ({len(body)} bytes)")
+                    except Exception as we:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)